frontend/package.json

```diff
@@ -10,6 +10,8 @@
   ],
   "scripts": {
     "clean": "rm -rf ./public/dist",
+    "clean-deps": "rm -rf ./**/**/node_modules",
+    "clean-install": "yarn clean-deps && yarn",
```

Comment:
I don't mind this, but we do have `clean-frontend.sh`. Do we need both?

Reply:
It's more just for convenience than anything. We could maybe just make these scripts reference the `clean-frontend.sh` script?

Reply:
It saves a couple of extra commands and is consistent with other helper yarn scripts. Currently, if you want to do a clean reinstall of frontend deps, and your current working dir is frontend (which at least for me, is almost always the case), you have to:

```
cd .. 
./clean-frontend.sh 
cd frontend 
yarn
``` 
whereas with these yarn scripts, you can do the same with just one command in your current working dir.

Reply:
Also, the `clean-frontend.sh` script does more than just remove frontend deps. These are focused specifically on just doing a clean reinstall of the frontend deps.

Reply:
I'm fine either way, but wouldn't the flow be:
```
console>./clean-frontend.sh
console>./build-frontend.sh    <--- does yarn install
```

Reply:
Yeah, and that is a production build as well, which might not be needed. These were really meant to just be used for dev purposes. Whenever I switch to an older release branch or I'm troubleshooting unusual build errors, I'll do a clean install of the frontend deps. These scripts were just meant to make that more convenient.

Reply:
I'd think it would be very rare that you need to do this. When I switch branches, `yarn install` almost always works. If you're having strange build errors, you probably want to clean everything. The reinstall of all the dependencies takes much longer than the rest of the build, so I don't really see a reason not to clean everything to build fresh.

If there is a common problem you're seeing switching branches, should we look at fixing that so this isn't needed?